src/ActionMonitor/Monitors/PreviewMonitor.php

```diff
@@ -109,29 +684,6 @@ public function post_to_preview_instance( $post_ID, $post ) {
 			return;
 		}
 
-		$last_sent_modified_time = Preview::get_last_sent_modified_time_by_post_id(
-			$parent_post_id
-		);
-
-		$last_sent_modified_time_unix = strtotime( $last_sent_modified_time );
-		$this_sent_modified_time_unix = strtotime( $post->post_modified );
-
-		$difference_between_last_modified_and_this_modified = 
-			$this_sent_modified_time_unix - $last_sent_modified_time_unix;
-
-		if (
-			$last_sent_modified_time &&
-			(
-				// if the last time was the same as this.
-				$last_sent_modified_time === $post->post_modified ||
-				// or the last time was within the last 5 seconds.
-				$difference_between_last_modified_and_this_modified < 5
-			)
-		) {
-			// we've already sent a webhook for this revision.
-			// return early to prevent extra builds.
-			return;
-		}
```

Comment:
@smthomas I removed this because it was causing webhooks to not fire sometimes. We should prefer more webhooks over the possibility of sending none. This logic is probably obsolete now anyway since we only send 1 webhook when loading the preview template now.